main/views.py

```diff
@@ -24,7 +24,10 @@ class PostList(ListView):
 
     context_object_name = 'posts'
     template_name = 'main/index.html'
-    queryset = Post.objects.filter(published_date__lte=timezone.now())
```

Comment:
Nope, here we, unfortunately, can't use it.
http://stackoverflow.com/questions/19707237/use-get-queryset-method-or-set-queryset-variable